### ISSUE TITLE
1.16.5-4.2.4-134 - Mining drill server bug

### ISSUE THREAD

**Issue by Acidglow** [BluSunrize/ImmersiveEngineering#4722]
### Description and Steps to Reproduce:
After adding a drill head and upgrades to a Mining Drill, they mysteriously disappear moments later.

Add a drill head and some miscellaneous upgrades to your Mining Drill on a multiplayer server. Remove drill head and then upgrades disappear, then put drill head back and it reappears.

Then dig a bit with the drill and put it back into the workbench, and now the upgrades are gone for good.

If you hold the drill in hand and open your inventory in creative mode, then the drill resets to its basic item without any drill head or upgrades installed anymore.

In version 130 the drill does not reset in creative but in 134 it does.

Here are some steps:
![1](https://user-images.githubusercontent.com/6869377/111376072-4cb84980-869f-11eb-8bfa-7e8f071706f5.png)
![2](https://user-images.githubusercontent.com/6869377/111376080-504bd080-869f-11eb-8128-8010803ffc11.png)
![3](https://user-images.githubusercontent.com/6869377/111376090-517cfd80-869f-11eb-9ed8-b61ca6080c16.png)
![4](https://user-images.githubusercontent.com/6869377/111376094-5346c100-869f-11eb-9566-799b9caeb2f5.png)
![5](https://user-images.githubusercontent.com/6869377/111376097-5477ee00-869f-11eb-81b6-ce50607cb543.png)
![6](https://user-images.githubusercontent.com/6869377/111376103-55a91b00-869f-11eb-94bc-7e045bdb4e8e.png)
![7](https://user-images.githubusercontent.com/6869377/111376106-56da4800-869f-11eb-9303-6ca48642c51c.png)
![8](https://user-images.githubusercontent.com/6869377/111376112-580b7500-869f-11eb-9058-22da447c7546.png)
![9](https://user-images.githubusercontent.com/6869377/111376121-59d53880-869f-11eb-8a2b-2e7de3a50725.png)

### Crashlog:
Server does not report any issues
F3+C=
[crash-2021-03-16_21.13.23-client.txt](https://github.com/BluSunrize/ImmersiveEngineering/files/6152001/crash-2021-03-16_21.13.23-client.txt)

### Versions & Modlist:
ImmersiveEngineering-1.16.5-4.2.4-134
jei-1.16.4-7.6.1.71 (Same issue without this mod)

**Comment by bluefire-fx** [comment]
I second this issue, experienced the same problem on my local server. It is easily reproducible.
## Versions
<ul>
<li>ImmersiveEngineering-1.16.5-5.0.0-137</li>
<ul>
<li>Forge-1.16.5-36.1.0</li>
<li>Forge-1.16.5-36.1.65</li>
</ul>
</ul>

**Comment by BluSunrize** [comment]
Do you only have these issues with a drill spawned in creative mode or can you reproduce it with a normally crafted one?

**Comment by bluefire-fx** [comment]
Created a testserver and cheated the ingots, treated wood and other raw materials but craftet onwars with stuff like machine casings from there on. Then all thats left to make the attachments disappear is going into creative AND open the Inventory. Thankfully shaders remain.

**Comment by DeadlySwede** [comment]
### 1.17.1 Update

Saw similar behavior on multiplayer IE 1.17 test server. Works fine in single player.

It appears that the items in the slots may still be there and this may be a visual glitch of some sort. If you click on the "empty" slots, the item will show up and you can drag it around. Sometimes dropping in one slot will cause another slot to disappear but it is really still there. Same thing happens with configuring the buzz saw in the workbench.

### Versions & Modlist

Minecraft 1.17.1
Forge 37.0.44
Immersive Engineering 1.17.1-5.0.2-pre.133
Journeymap 5.7.3rc3
Just Enough Items 8.0.0.14

**Comment by M1chalS** [comment]
# 1.19.2 Update
I still have the same issue on my private server on 1.19.2...

I made drill in survival mode and put head and all the upgrades on it. Then when I go creative drill is still fine but when I go back to survival drill head and all upgrades disappear. Only chroma remains. Items don't reappear at any case from my observation. 

# Version
Minecraft 1.19.2
Forge 43.1.55
ImmersiveEngineering-1.19.2-9.0.0-153

**Comment by BluSunrize** [comment]
Yeah and this continues to be a Forge issue that we can't do anything about.
